### PR TITLE
Greek translation update 13/10/2019

### PR DESCRIPTION
**Pull request by MCelliotG**

### CODE REVIEW

File: po/el.po [hunk 1]
```diff
@@ -7751,7 +7751,7 @@ msgid "Show menu paths"
 msgstr "Εμφάνιση διαδρομών μενού"
 
 msgid "Show message when recording starts"
-msgstr "Εμφάνιση μηνύματος στην εκκίνηση της εγραφής"
+msgstr "Εμφάνιση μηνύματος στην έναρξη της εγραφής"
```
Comment (nautilus7) [inline po/el.po:7754]: It should be "εγγραφής" with double "γ".

**Comment by nautilus7** [comment]
Also in line 5131 you forgot to fix another string containing "downmix".

"Multi channel downmix" should become "Αποκωδικοποίηση & μίξη πολυκάναλων ήχων" or Αποκωδικοποίηση & μίξη πολυκάναλων ροών ήχου".

**Comment by nautilus7** [comment]
And another one I have found: In line 11191, "pass" should be "πέρασμα" or something similar. It refers to service scanning, where in some cases we can do one or more scanning passes.

File: po/el.po (hunk 1 above)
Comment (MCelliotG) [reply po/el.po:7754]: Yes, it was done the last minute, will be added after the next refresh.

**Pull request merged**